modules/base/mw.cx.SiteMapper.js

```
@@ -23,6 +23,7 @@ mw.cx.SiteMapper = class {
 
 		const siteMapperConfig = Object.assign( {}, config, overrides );
 		this.siteTemplates = siteMapperConfig.SiteTemplates;
+		this.SiteTemplates_mdwiki = siteMapperConfig.SiteTemplates_mdwiki;
```

ESLint

[error] Identifier 'SiteTemplates_mdwiki' is not in camel case.

(camelcase)